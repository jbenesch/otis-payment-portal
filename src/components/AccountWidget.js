import React from 'react'
import { Row, Column } from 'hedron'
import styled from 'styled-components'
import {
  Button,
  ButtonGroup,
  Colors,
  Icon,
  Intent,
  Tag
} from '@blueprintjs/core'
import Widget from '../components/Widget'

const WidgetTitle = styled.h5`
  color: ${Colors.LIGHT_GRAY5};
`

const AccountWidget = () => (
  <Widget>
    <Row>
      <Column sm={7} style={{ paddingBottom: 5, paddingLeft: 10 }}>
        <WidgetTitle>My Accounts</WidgetTitle>
      </Column>
      <Column
        sm={1}
        style={{ paddingBottom: 5, paddingRight: 0, textAlign: 'right' }}
      >
        <Icon icon="search" iconSize={Icon.SIZE_LARGE} />
      </Column>
      <Column
        sm={4}
        style={{
          paddingBottom: 0,
          paddingTop: 15,
          paddingRight: 10,
          textAlign: 'right'
        }}
      >
        <Tag
          interactive
          large
          minimal
          round
          style={{ color: Colors.LIGHT_GRAY1 }}
        >
          <Icon icon="filter-list" style={{ marginRight: 10 }} />
          Filter by
        </Tag>
      </Column>
    </Row>
    <table className="pt-html-table" style={{ width: '100%' }}>
      <tbody>
        <tr>
          <td style={{ paddingTop: 25 }}>
            Blueprint - CSS framework and UI toolkit
          </td>
          <td style={{ paddingTop: 25, textAlign: 'right' }}>
            <ButtonGroup>
              <Button icon="tick" intent={Intent.SUCCESS} />
              <Button icon="cog" rightIcon="chevron-down" />
            </ButtonGroup>
          </td>
        </tr>
        <tr>
          <td>TSLint - Static analysis linter for TypeScript</td>
          <td style={{ textAlign: 'right' }}>
            <ButtonGroup>
              <ButtonGroup>
                <Button icon="tick" />
                <Button icon="cog" rightIcon="chevron-down" />
              </ButtonGroup>
            </ButtonGroup>
          </td>
        </tr>
        <tr>
          <td>Plottable - Composable charting library built on top of D3</td>
          <td style={{ textAlign: 'right' }}>
            <ButtonGroup>
              <ButtonGroup>
                <Button icon="tick" />
                <Button icon="cog" rightIcon="chevron-down" />
              </ButtonGroup>
            </ButtonGroup>
          </td>
        </tr>
        <tr>
          <td>TSLint - Static analysis linter for TypeScript</td>
          <td style={{ textAlign: 'right' }}>
            <ButtonGroup>
              <ButtonGroup>
                <Button icon="tick" />
                <Button icon="cog" rightIcon="chevron-down" />
              </ButtonGroup>
            </ButtonGroup>
          </td>
        </tr>
        <tr>
          <td>Plottable - Composable charting library built on top of D3</td>
          <td style={{ textAlign: 'right' }}>
            <ButtonGroup>
              <ButtonGroup>
                <Button icon="tick" />
                <Button icon="cog" rightIcon="chevron-down" />
              </ButtonGroup>
            </ButtonGroup>
          </td>
        </tr>
      </tbody>
    </table>
  </Widget>
)

AccountWidget.propTypes = {}

AccountWidget.defaultProps = {}

export default AccountWidget
