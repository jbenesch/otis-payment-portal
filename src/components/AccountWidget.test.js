import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import AccountWidget from './AccountWidget'

it('matches a previous snapshot', () => {
  const component = renderer.create(<AccountWidget />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('should render the account widget', () => {
  const wrapper = shallow(<AccountWidget />)
  expect(wrapper).toMatchSnapshot()
})
