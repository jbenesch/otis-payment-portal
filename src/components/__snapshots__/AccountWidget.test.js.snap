// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches a previous snapshot 1`] = `
<div
  className="sc-ifAKCX cqZWro"
>
  <section
    className="sc-bwzfXH irrVft"
  >
    <div
      className="sc-htpNat fpNCPr"
      style={
        Object {
          "paddingBottom": 5,
          "paddingLeft": 10,
        }
      }
    >
      <h5
        className="sc-EHOje dQOHAj"
      >
        My Accounts
      </h5>
    </div>
    <div
      className="sc-htpNat klRRny"
      style={
        Object {
          "paddingBottom": 5,
          "paddingRight": 0,
          "textAlign": "right",
        }
      }
    >
      <svg
        className="pt-icon"
        data-icon="search"
        height={20}
        style={Object {}}
        viewBox="0 0 20 20"
        width={20}
      >
        <title>
          search
        </title>
        <path
          d="M19.56 17.44l-4.94-4.94A8.004 8.004 0 0 0 16 8c0-4.42-3.58-8-8-8S0 3.58 0 8s3.58 8 8 8c1.67 0 3.21-.51 4.5-1.38l4.94 4.94a1.498 1.498 0 1 0 2.12-2.12zM8 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6z"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <div
      className="sc-htpNat kLfyZ"
      style={
        Object {
          "paddingBottom": 0,
          "paddingRight": 10,
          "paddingTop": 15,
          "textAlign": "right",
        }
      }
    >
      <span
        className="pt-tag pt-interactive pt-large pt-minimal pt-round"
        style={
          Object {
            "color": "#CED9E0",
          }
        }
      >
        <svg
          className="pt-icon"
          data-icon="filter-list"
          height={16}
          style={
            Object {
              "marginRight": 10,
            }
          }
          viewBox="0 0 16 16"
          width={16}
        >
          <title>
            filter-list
          </title>
          <path
            d="M9 8c0 .55.45 1 1 1h5c.55 0 1-.45 1-1s-.45-1-1-1h-5c-.55 0-1 .45-1 1zm3-6c0-.55-.45-1-1-1H1a1.003 1.003 0 0 0-.71 1.71L4 6.41V12a1.003 1.003 0 0 0 1.71.71l2-2c.18-.18.29-.43.29-.71V6.41l3.71-3.71c.18-.17.29-.42.29-.7zm3 8h-5c-.55 0-1 .45-1 1s.45 1 1 1h5c.55 0 1-.45 1-1s-.45-1-1-1zm0 3h-5c-.55 0-1 .45-1 1s.45 1 1 1h5c.55 0 1-.45 1-1s-.45-1-1-1z"
            fillRule="evenodd"
          />
        </svg>
        Filter by
      </span>
    </div>
  </section>
  <table
    className="pt-html-table"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <tbody>
      <tr>
        <td
          style={
            Object {
              "paddingTop": 25,
            }
          }
        >
          Blueprint - CSS framework and UI toolkit
        </td>
        <td
          style={
            Object {
              "paddingTop": 25,
              "textAlign": "right",
            }
          }
        >
          <div
            className="pt-button-group"
          >
            <button
              className="pt-button pt-intent-success"
              disabled={undefined}
              onClick={undefined}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              type="button"
            >
              <svg
                className="pt-icon"
                data-icon="tick"
                height={16}
                style={Object {}}
                viewBox="0 0 16 16"
                width={16}
              >
                <title>
                  tick
                </title>
                <path
                  d="M14 3c-.28 0-.53.11-.71.29L6 10.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l8-8A1.003 1.003 0 0 0 14 3z"
                  fillRule="evenodd"
                />
              </svg>
            </button>
            <button
              className="pt-button"
              disabled={undefined}
              onClick={undefined}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              type="button"
            >
              <svg
                className="pt-icon"
                data-icon="cog"
                height={16}
                style={Object {}}
                viewBox="0 0 16 16"
                width={16}
              >
                <title>
                  cog
                </title>
                <path
                  d="M15.19 6.39h-1.85c-.11-.37-.27-.71-.45-1.04l1.36-1.36c.31-.31.31-.82 0-1.13l-1.13-1.13a.803.803 0 0 0-1.13 0l-1.36 1.36c-.33-.17-.67-.33-1.04-.44V.79c0-.44-.36-.8-.8-.8h-1.6c-.44 0-.8.36-.8.8v1.86c-.39.12-.75.28-1.1.47l-1.3-1.3c-.3-.3-.79-.3-1.09 0L1.82 2.91c-.3.3-.3.79 0 1.09l1.3 1.3c-.2.34-.36.7-.48 1.09H.79c-.44 0-.8.36-.8.8v1.6c0 .44.36.8.8.8h1.85c.11.37.27.71.45 1.04l-1.36 1.36c-.31.31-.31.82 0 1.13l1.13 1.13c.31.31.82.31 1.13 0l1.36-1.36c.33.18.67.33 1.04.44v1.86c0 .44.36.8.8.8h1.6c.44 0 .8-.36.8-.8v-1.86c.39-.12.75-.28 1.1-.47l1.3 1.3c.3.3.79.3 1.09 0l1.09-1.09c.3-.3.3-.79 0-1.09l-1.3-1.3c.19-.35.36-.71.48-1.1h1.85c.44 0 .8-.36.8-.8v-1.6a.816.816 0 0 0-.81-.79zm-7.2 4.6c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                  fillRule="evenodd"
                />
              </svg>
              <svg
                className="pt-icon"
                data-icon="chevron-down"
                height={16}
                style={Object {}}
                viewBox="0 0 16 16"
                width={16}
              >
                <title>
                  chevron-down
                </title>
                <path
                  d="M12 5c-.28 0-.53.11-.71.29L8 8.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l4-4A1.003 1.003 0 0 0 12 5z"
                  fillRule="evenodd"
                />
              </svg>
            </button>
          </div>
        </td>
      </tr>
      <tr>
        <td>
          TSLint - Static analysis linter for TypeScript
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <div
            className="pt-button-group"
          >
            <div
              className="pt-button-group"
            >
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="tick"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    tick
                  </title>
                  <path
                    d="M14 3c-.28 0-.53.11-.71.29L6 10.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l8-8A1.003 1.003 0 0 0 14 3z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="cog"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    cog
                  </title>
                  <path
                    d="M15.19 6.39h-1.85c-.11-.37-.27-.71-.45-1.04l1.36-1.36c.31-.31.31-.82 0-1.13l-1.13-1.13a.803.803 0 0 0-1.13 0l-1.36 1.36c-.33-.17-.67-.33-1.04-.44V.79c0-.44-.36-.8-.8-.8h-1.6c-.44 0-.8.36-.8.8v1.86c-.39.12-.75.28-1.1.47l-1.3-1.3c-.3-.3-.79-.3-1.09 0L1.82 2.91c-.3.3-.3.79 0 1.09l1.3 1.3c-.2.34-.36.7-.48 1.09H.79c-.44 0-.8.36-.8.8v1.6c0 .44.36.8.8.8h1.85c.11.37.27.71.45 1.04l-1.36 1.36c-.31.31-.31.82 0 1.13l1.13 1.13c.31.31.82.31 1.13 0l1.36-1.36c.33.18.67.33 1.04.44v1.86c0 .44.36.8.8.8h1.6c.44 0 .8-.36.8-.8v-1.86c.39-.12.75-.28 1.1-.47l1.3 1.3c.3.3.79.3 1.09 0l1.09-1.09c.3-.3.3-.79 0-1.09l-1.3-1.3c.19-.35.36-.71.48-1.1h1.85c.44 0 .8-.36.8-.8v-1.6a.816.816 0 0 0-.81-.79zm-7.2 4.6c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  className="pt-icon"
                  data-icon="chevron-down"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    chevron-down
                  </title>
                  <path
                    d="M12 5c-.28 0-.53.11-.71.29L8 8.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l4-4A1.003 1.003 0 0 0 12 5z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
            </div>
          </div>
        </td>
      </tr>
      <tr>
        <td>
          Plottable - Composable charting library built on top of D3
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <div
            className="pt-button-group"
          >
            <div
              className="pt-button-group"
            >
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="tick"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    tick
                  </title>
                  <path
                    d="M14 3c-.28 0-.53.11-.71.29L6 10.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l8-8A1.003 1.003 0 0 0 14 3z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="cog"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    cog
                  </title>
                  <path
                    d="M15.19 6.39h-1.85c-.11-.37-.27-.71-.45-1.04l1.36-1.36c.31-.31.31-.82 0-1.13l-1.13-1.13a.803.803 0 0 0-1.13 0l-1.36 1.36c-.33-.17-.67-.33-1.04-.44V.79c0-.44-.36-.8-.8-.8h-1.6c-.44 0-.8.36-.8.8v1.86c-.39.12-.75.28-1.1.47l-1.3-1.3c-.3-.3-.79-.3-1.09 0L1.82 2.91c-.3.3-.3.79 0 1.09l1.3 1.3c-.2.34-.36.7-.48 1.09H.79c-.44 0-.8.36-.8.8v1.6c0 .44.36.8.8.8h1.85c.11.37.27.71.45 1.04l-1.36 1.36c-.31.31-.31.82 0 1.13l1.13 1.13c.31.31.82.31 1.13 0l1.36-1.36c.33.18.67.33 1.04.44v1.86c0 .44.36.8.8.8h1.6c.44 0 .8-.36.8-.8v-1.86c.39-.12.75-.28 1.1-.47l1.3 1.3c.3.3.79.3 1.09 0l1.09-1.09c.3-.3.3-.79 0-1.09l-1.3-1.3c.19-.35.36-.71.48-1.1h1.85c.44 0 .8-.36.8-.8v-1.6a.816.816 0 0 0-.81-.79zm-7.2 4.6c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  className="pt-icon"
                  data-icon="chevron-down"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    chevron-down
                  </title>
                  <path
                    d="M12 5c-.28 0-.53.11-.71.29L8 8.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l4-4A1.003 1.003 0 0 0 12 5z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
            </div>
          </div>
        </td>
      </tr>
      <tr>
        <td>
          TSLint - Static analysis linter for TypeScript
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <div
            className="pt-button-group"
          >
            <div
              className="pt-button-group"
            >
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="tick"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    tick
                  </title>
                  <path
                    d="M14 3c-.28 0-.53.11-.71.29L6 10.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l8-8A1.003 1.003 0 0 0 14 3z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="cog"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    cog
                  </title>
                  <path
                    d="M15.19 6.39h-1.85c-.11-.37-.27-.71-.45-1.04l1.36-1.36c.31-.31.31-.82 0-1.13l-1.13-1.13a.803.803 0 0 0-1.13 0l-1.36 1.36c-.33-.17-.67-.33-1.04-.44V.79c0-.44-.36-.8-.8-.8h-1.6c-.44 0-.8.36-.8.8v1.86c-.39.12-.75.28-1.1.47l-1.3-1.3c-.3-.3-.79-.3-1.09 0L1.82 2.91c-.3.3-.3.79 0 1.09l1.3 1.3c-.2.34-.36.7-.48 1.09H.79c-.44 0-.8.36-.8.8v1.6c0 .44.36.8.8.8h1.85c.11.37.27.71.45 1.04l-1.36 1.36c-.31.31-.31.82 0 1.13l1.13 1.13c.31.31.82.31 1.13 0l1.36-1.36c.33.18.67.33 1.04.44v1.86c0 .44.36.8.8.8h1.6c.44 0 .8-.36.8-.8v-1.86c.39-.12.75-.28 1.1-.47l1.3 1.3c.3.3.79.3 1.09 0l1.09-1.09c.3-.3.3-.79 0-1.09l-1.3-1.3c.19-.35.36-.71.48-1.1h1.85c.44 0 .8-.36.8-.8v-1.6a.816.816 0 0 0-.81-.79zm-7.2 4.6c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  className="pt-icon"
                  data-icon="chevron-down"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    chevron-down
                  </title>
                  <path
                    d="M12 5c-.28 0-.53.11-.71.29L8 8.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l4-4A1.003 1.003 0 0 0 12 5z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
            </div>
          </div>
        </td>
      </tr>
      <tr>
        <td>
          Plottable - Composable charting library built on top of D3
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <div
            className="pt-button-group"
          >
            <div
              className="pt-button-group"
            >
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="tick"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    tick
                  </title>
                  <path
                    d="M14 3c-.28 0-.53.11-.71.29L6 10.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l8-8A1.003 1.003 0 0 0 14 3z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
              <button
                className="pt-button"
                disabled={undefined}
                onClick={undefined}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                type="button"
              >
                <svg
                  className="pt-icon"
                  data-icon="cog"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    cog
                  </title>
                  <path
                    d="M15.19 6.39h-1.85c-.11-.37-.27-.71-.45-1.04l1.36-1.36c.31-.31.31-.82 0-1.13l-1.13-1.13a.803.803 0 0 0-1.13 0l-1.36 1.36c-.33-.17-.67-.33-1.04-.44V.79c0-.44-.36-.8-.8-.8h-1.6c-.44 0-.8.36-.8.8v1.86c-.39.12-.75.28-1.1.47l-1.3-1.3c-.3-.3-.79-.3-1.09 0L1.82 2.91c-.3.3-.3.79 0 1.09l1.3 1.3c-.2.34-.36.7-.48 1.09H.79c-.44 0-.8.36-.8.8v1.6c0 .44.36.8.8.8h1.85c.11.37.27.71.45 1.04l-1.36 1.36c-.31.31-.31.82 0 1.13l1.13 1.13c.31.31.82.31 1.13 0l1.36-1.36c.33.18.67.33 1.04.44v1.86c0 .44.36.8.8.8h1.6c.44 0 .8-.36.8-.8v-1.86c.39-.12.75-.28 1.1-.47l1.3 1.3c.3.3.79.3 1.09 0l1.09-1.09c.3-.3.3-.79 0-1.09l-1.3-1.3c.19-.35.36-.71.48-1.1h1.85c.44 0 .8-.36.8-.8v-1.6a.816.816 0 0 0-.81-.79zm-7.2 4.6c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  className="pt-icon"
                  data-icon="chevron-down"
                  height={16}
                  style={Object {}}
                  viewBox="0 0 16 16"
                  width={16}
                >
                  <title>
                    chevron-down
                  </title>
                  <path
                    d="M12 5c-.28 0-.53.11-.71.29L8 8.59l-3.29-3.3a1.003 1.003 0 0 0-1.42 1.42l4 4c.18.18.43.29.71.29s.53-.11.71-.29l4-4A1.003 1.003 0 0 0 12 5z"
                    fillRule="evenodd"
                  />
                </svg>
              </button>
            </div>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`should render the account widget 1`] = `
<styled.div>
  <Styled(RowContainer)>
    <Breakpoints
      sm={7}
      style={
        Object {
          "paddingBottom": 5,
          "paddingLeft": 10,
        }
      }
    >
      <styled.h5>
        My Accounts
      </styled.h5>
    </Breakpoints>
    <Breakpoints
      sm={1}
      style={
        Object {
          "paddingBottom": 5,
          "paddingRight": 0,
          "textAlign": "right",
        }
      }
    >
      <Blueprint2.Icon
        icon="search"
        iconSize={20}
      />
    </Breakpoints>
    <Breakpoints
      sm={4}
      style={
        Object {
          "paddingBottom": 0,
          "paddingRight": 10,
          "paddingTop": 15,
          "textAlign": "right",
        }
      }
    >
      <Blueprint2.Tag
        interactive={true}
        large={true}
        minimal={true}
        round={true}
        style={
          Object {
            "color": "#CED9E0",
          }
        }
      >
        <Blueprint2.Icon
          icon="filter-list"
          style={
            Object {
              "marginRight": 10,
            }
          }
        />
        Filter by
      </Blueprint2.Tag>
    </Breakpoints>
  </Styled(RowContainer)>
  <table
    className="pt-html-table"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <tbody>
      <tr>
        <td
          style={
            Object {
              "paddingTop": 25,
            }
          }
        >
          Blueprint - CSS framework and UI toolkit
        </td>
        <td
          style={
            Object {
              "paddingTop": 25,
              "textAlign": "right",
            }
          }
        >
          <Blueprint2.ButtonGroup>
            <Blueprint2.Button
              icon="tick"
              intent="success"
            />
            <Blueprint2.Button
              icon="cog"
              rightIcon="chevron-down"
            />
          </Blueprint2.ButtonGroup>
        </td>
      </tr>
      <tr>
        <td>
          TSLint - Static analysis linter for TypeScript
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <Blueprint2.ButtonGroup>
            <Blueprint2.ButtonGroup>
              <Blueprint2.Button
                icon="tick"
              />
              <Blueprint2.Button
                icon="cog"
                rightIcon="chevron-down"
              />
            </Blueprint2.ButtonGroup>
          </Blueprint2.ButtonGroup>
        </td>
      </tr>
      <tr>
        <td>
          Plottable - Composable charting library built on top of D3
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <Blueprint2.ButtonGroup>
            <Blueprint2.ButtonGroup>
              <Blueprint2.Button
                icon="tick"
              />
              <Blueprint2.Button
                icon="cog"
                rightIcon="chevron-down"
              />
            </Blueprint2.ButtonGroup>
          </Blueprint2.ButtonGroup>
        </td>
      </tr>
      <tr>
        <td>
          TSLint - Static analysis linter for TypeScript
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <Blueprint2.ButtonGroup>
            <Blueprint2.ButtonGroup>
              <Blueprint2.Button
                icon="tick"
              />
              <Blueprint2.Button
                icon="cog"
                rightIcon="chevron-down"
              />
            </Blueprint2.ButtonGroup>
          </Blueprint2.ButtonGroup>
        </td>
      </tr>
      <tr>
        <td>
          Plottable - Composable charting library built on top of D3
        </td>
        <td
          style={
            Object {
              "textAlign": "right",
            }
          }
        >
          <Blueprint2.ButtonGroup>
            <Blueprint2.ButtonGroup>
              <Blueprint2.Button
                icon="tick"
              />
              <Blueprint2.Button
                icon="cog"
                rightIcon="chevron-down"
              />
            </Blueprint2.ButtonGroup>
          </Blueprint2.ButtonGroup>
        </td>
      </tr>
    </tbody>
  </table>
</styled.div>
`;
